src/crab/ebpf_domain.cpp

```
@@ -264,99 +263,80 @@ bool EbpfDomain::get_map_fd_range(const Reg& map_fd_reg, int32_t* start_fd, int3
 }
 
 // All maps in the range must have the same type for us to use it.
-std::optional<uint32_t> EbpfDomain::get_map_type(const Reg& map_fd_reg, const ebpf_platform_t& platform) const {
+std::optional<uint32_t> EbpfDomain::get_map_type(const Reg& map_fd_reg, const AnalysisContext& context) const {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)